crates/matrix-store-key/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "matrix-store-key"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+blake3 = "1.3.1"
+chacha20poly1305 = { version = "0.9.0", features = ["std"] }
+displaydoc = "0.2"
+hmac = "0.12.1"
+pbkdf2 = "0.10.1"
+rand = "0.8.4"
+serde = { version = "1.0.126", features = ["derive"] }
+serde_json = "1.0.64"
```

Comment:
so... considering that nothing in the actual API is bound to serde_json (other than the Error-Type) and we only use it to generate `&[u8]`, is this the right time to open the question, whether we want to move away from json here and [instead go for any of the more performant serde](https://github.com/djkoloski/rust_serialization_benchmark) - e.g. [speedy](https://github.com/koute/speedy)?

Reply:
Not sure, Matrix heavily depends on JSON and in turn almost all of our crates will depend on serde/serde_json because of that. That in turn creates an aversion towards using yet another serialization library.